### PR TITLE
[iOS]Upload iOS playground

### PR DESCRIPTION
**Pull request by wrmswindmill**

### CODE REVIEW

**Comment by janpio** [comment]
Should the `Pods` directory be included in the codebase?

**Comment by wrmswindmill** [comment]
> Should the `Pods` directory be included in the codebase?

**Comment by wrmswindmill** [comment]
In this case, we don’t need include `Pods` directory, I will update the `.gitignore` and remove the  `Pods` directory.

**Comment by janpio** [comment]
Much better, thanks! 🚀

**Comment by YorkShen** [comment]
The code looks like good, but there is a problem: **All the git commit history is missing**.  

Using `git subtree split` in https://github.com/apache/incubator-weex will give you the full git commit history.

**Comment by wrmswindmill** [comment]
thanks for your advice,and I have added all the missing commit history now.

File: .gitignore [hunk 1]
```diff
@@ -0,0 +1,2 @@
+# CocoaPods
+ios/Pods/
```
Comment (janpio) [inline .gitignore]: You probably want that in a `.gitignore` in `ios/` itself instead of in `/`.

**Review by YorkShen — APPROVED** [review]
Looks good to me, thanks for your contribution.

Next, we shall delete the corresponding code in https://github.com/apache/incubator-weex .

**Pull request merged**